src/components/toggle/daikin-toggle.ts

```diff
@@ -0,0 +1,152 @@
+import { cva } from "class-variance-authority";
+import { css, html, LitElement, unsafeCSS, type PropertyValues } from "lit";
+import { customElement, property, query } from "lit/decorators.js";
+import tailwindStyles from "../../tailwind.css?inline";
+import type { MergeVariantProps } from "../../type-utils";
+
+const cvaToggle = cva(
+  [
+    "appearance-none",
+    "cursor-pointer",
+    "relative",
+    "bg-daikinNeutral-200",
+    "rounded-3xl",
+    "transition-color",
+    "duration-300",
+    "disabled:bg-daikinNeutral-200",
+
+    "before:content-['']",
+    "before:absolute",
+    "before:rounded-full",
+    "before:transition",
+    "before:disabled:bg-daikinNeutral-400",
+  ],
+  {
+    variants: {
+      size: {
+        default: [
+          "w-[51px]",
+          "h-[31px]",
+          "checked:bg-daikinBlue-500",
+
+          "before:h-[27px]",
+          "before:w-[27px]",
+          "before:bg-white",
+          "before:top-[2px]",
+          "before:left-[2px]",
+          "before:checked:translate-x-5",
+        ],
+        small: [
+          "w-8",
+          "h-[14px]",
+          "checked:bg-daikinBlue-50",
+
+          "before:h-5",
+          "before:w-5",
+          "before:top-[-3px]",
+          "before:bg-daikinNeutral-600",
+          "before:checked:bg-daikinBlue-500",
+          "before:checked:translate-x-3",
+        ],
+      },
+    },
+  }
+);
+
+type ToggleVariantProps = MergeVariantProps<typeof cvaToggle>;
+
+/**
+ * Primary UI component for user interaction
```

Comment:
I would like to add [lit-analyzer friendly JSDoc](https://github.com/runem/lit-analyzer/blob/master/docs/readme/jsdoc.md).
(`@element`, `@attr` and `@prop` are not needed as they're inferred from individual prop definitions.)

```suggestion
 * A toggle switch component.
 *
 * @fires change - Emitted when the toggle switch is toggled.
```